kstars/fitsviewer/fitsbahtinovdetector.cpp

```diff
@@ -0,0 +1,445 @@
+/***************************************************************************
+                          fitsbahtinovdetector.cpp  -  FITS Image
+                             -------------------
+    begin                : Sat March 28 2020
+    copyright            : (C) 2004 by Jasem Mutlaq, (C) 2020 by Eric Dejouhanet
+    email                : eric.dejouhanet@gmail.com
```

Comment:
Copyright should be yours.

Reply:
I will update the copyright for these files.